scripts/setup.sh

```diff
@@ -1,14 +1,50 @@
 #!/usr/bin/env bash
-
 set -ex
 
-apt-get update
-source /opt/ros/$ROS_DISTRO/setup.sh
+if [[ -z "$ROS_DISTRO"  ]]
+then
+		#timezone is required for ROS Installation.
+		if [ ! -f "/etc/timezone" ];
+		then
+			export TZ="US/Pacific"
```

Comment:
Any thoughts on defaulting to some timezone?

Reply:
will move this step into github actions.  If the customer doesn't have this set, they would get prompted in the middle of ROS installation to set it.